libr/socket/run.c

```diff
@@ -911,6 +916,22 @@ R_API int r_run_config_env(RRunProfile *p) {
 	return 0;
 }
 
+static void r_run_call1(void* fcn, void* arg1) {
```

Comment:
May be make those inline and move to some header? (Along with adding more parameters count).